chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/watcher/outgoing/ChaosMonkeyRestTemplateWatcherIntegrationTest.java

```diff
@@ -63,22 +63,23 @@ class ExceptionAssaultIntegrationTest {
 
         @Test
         public void testRestTemplateExceptionAssault() {
-            assertThatThrownBy(() -> this.demoRestTemplateService.callWithRestTemplate()).hasMessage("500 INTERNAL_SERVER_ERROR");
+            assertThatThrownBy(() -> demoRestTemplateService.callWithRestTemplate()).hasMessage("500 INTERNAL_SERVER_ERROR");
         }
     }
 
-    @Slf4j
     @SpringBootTest(properties = {"chaos.monkey.enabled=true", "chaos.monkey.watcher.rest-template=true", "chaos.monkey.assaults.latency-active=true",
             "chaos.monkey.test.rest-template.time-out=20"}, classes = {ChaosDemoApplication.class})
     @ActiveProfiles("chaos-monkey")
-    static class LatencyAssaultIntegrationTest {
+    @Nested
+    @Disabled
+    class LatencyAssaultIntegrationTest {
```

Comment:
> deactivate the unused test instead of blocking it incorrectly

I'm not sure I understand what you mean with this commit message and why you now disabled the test entirely. However, I do agree that the test is kind of weird at this time because it fails on my local machine on the `main` branch. Shouldn't we try to fix the test instead of disabling it?